### ISSUE TITLE
[Bug][SF][2.6.0][FlexibleColumnLayout][A11y][Expected reply Time: ASAP] Resizing/dragging action of columns missing single pointer activation for mouse and touch users

### ISSUE THREAD

**Issue by wilson-xu-sap** [SAP/ui5-webcomponents#10668]
### Describe the bug

The Resizing/dragging action of columns missing single pointer activation for mouse and touch users.

### Isolated Example

https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/layouts-floorplans-flexiblecolumnlayout--docs

### Expected Behaviour

A single pointer activation mechanism must be available so mouse and touch users don't have to perform the dragging operation.

### UI5 Web Components for React Version

2.6.0

### UI5 Web Components Version

2.6.0

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by kskondov** [comment]
Hello @SAP/ui5-webcomponents-topic-p can you take a look at this request to add a single point activation for dragging and resizing

**Comment by dobrinyonkov** [comment]
Hi @wilson-xu-sap,

Thanks for reporting.

We've seen this reported before, and OS features like Windows Click Lock, On-Screen Keyboard, and iOS Assistive Touch often help.

**Windows Click Lock & MacOS Drag Lock:** Enables drag mode with a single click and move, then a second click to drop.
**On-Screen Keyboard:** Allows using shortcuts like Ctrl+Right for drag actions.
**iOS Assistive Touch:** (Apple iOS, iPadOS) Provides a Hold and Drag option for tap-based dragging.

Have you tried any of these and if so did you face any issues? Could you check if these work for your case? If not, let us know what issues you’re facing and why they aren’t sufficient.

Thanks and Best Regards,
Dobrin

**Comment by wilson-xu-sap** [comment]
Hi @dobrinyonkov I am OK with that since I am just a gateway of this issue. I will let a11y team test this using your method again. Thanks.

**Comment by dobrinyonkov** [comment]
Great! Let us know of any issues. 

Regards,
Dobrin
